<launch>
  <node name="plane_seg" pkg="plane_seg_ros" type="plane_seg_ros" output="screen">
    <remap from="/plane_seg/pose" to="/state_estimator/pose_in_odom"/>
    <param name="/run_test_program" value="false" />
  </node>
</launch>
